src/benchmark/file_based_benchmark.cpp

```diff
@@ -25,21 +25,24 @@ int main(int argc, char* argv[]) {
   cli_options.add_options()
       ("table_path", "Directory containing the Tables as csv, tbl or binary files. CSV files require meta-files, see csv_meta.hpp or any *.csv.json file.", cxxopts::value<std::string>()->default_value(".")) // NOLINT
       ("query_path", "A specific .sql file or a directory containing .sql files", cxxopts::value<std::string>()->default_value(".")) // NOLINT
-      ("q,queries", "Subset of queries to run as a comma separated list", cxxopts::value<std::string>()->default_value("all")); // NOLINT
+      ("q,queries", "Subset of queries to run as a comma separated list", cxxopts::value<std::string>()->default_value("all")) // NOLINT
+      ("excluded_queries", "Non-queries to exlude as a comma separated list", cxxopts::value<std::string>()->default_value("none")); // NOLINT
```

Comment:
We could delete the files beforehand.

Reply:
Or use --queries?

Reply:
That doesn't work right now because in the constructor of the FileBasedQueryGenerator all SQL files in the given directory are checked for valid SQL in line 65 and the hsql parser doesn't support indexes right now. Therefore, an error is thrown.